docs/reference/latency.md

```diff
@@ -0,0 +1,96 @@
+After mining is successful, the new block candidate is added to the target chain.
+
+## Single chain transactions
+
+On average, single chain transactions take approximately 45 seconds.
+However, transaction latency—defined as the time between submitting a transaction to a node `/send` endpoint and seeing its result with the `/poll` endpoint—can take much longer than 45 seconds, in particular, if there are mining delays.
+
+To give you better insight into how transactions are executed, the following example describes the workflow for a single chain transaction.
+You start a transaction by submitting a formatted request to the `/send` endpoint on a node.
+The code provided by the `/send` endpoint performs some simple initial checks that validate the transaction has a reasonably high chance of being executed successfully. 
+These initial checks ensure that you learn if a transaction is invalid as soon as possible. 
+If the transaction passes the initial checks, the `/send` process adds the transaction to the mempool for the local node. 
+The transaction is then gossiped to remote nodes, and, eventually, one of the nodes is selected to mine the next block. 
+After the mined block is added to its target chain, the block with the transaction result is synchronized on the local node, and a request to the /poll endpoint returns the transaction result.
+
+Under normal load, the initial checks should take less than a millisecond for a single transaction.
+However, if a node in under heavy load—for example, because the PactService is contending with a high volume of `/local` or `/send` calls or because the node is catching up—you might experience delays in transaction processing. 
+It's worth noting that if a node is too busy, it will always prioritize staying caught up over sending transactions.
+
+The propagation of transactions in the mempool to other nodes takes, on average, 10 seconds, but it can take longer if there are networking delays, such as increased network traffic or slow network connections between nodes.
+Block propagation back to the local node is usually a matter of seconds.
+
+## Cross-chain transfers
+
+As discussed in [Transaction format and flow](/smart-contracts/transactions), a cross-chain transfer is essentially two separate but related transactions.
+The time it takes to complete a cross-chain transfer can depend on several factors, including the physical distance between the chain nodes in the network, network congestion, and the synchronization of state between the chains involved.
+
+For two distant chains—representing the worst case—a cross-chain transfer is expected to take the following time to complete:
+
+- The initial transaction is a exec transaction with an average time from submission to completion of 45 seconds.
+- Two or more blocks must be propagated to the target chain so that the source and target chain share the same view of state, with an average time of 30 seconds per block.
```

Comment:
```suggestion
- With the current 20-chain graph, two blocks at most must be propagated to the target chain so that the source and target chain share the same view of state, with an average time of 30 seconds per block. This will increase as more chains are added, and will depend on the distance between the chains.
```

Reply:
The diameter of the graph is 2, so that's the max. It could also be 1.